packages/cli/commands/lint.js

```diff
@@ -1,7 +1,7 @@
 const { lint } = require('@hubspot/cli-lib/validate');
 const { printHublValidationResult } = require('../lib/hublValidate');
 const { logger } = require('@hubspot/cli-lib/logger');
-const { logErrorInstance } = require('../lib/errorHandlers/standardErrors');
+const { logErrorInstance } = require('@hubspot/cli-lib/errorHandlers');
```

Comment:
Should this import be coming from within the cli now?

Reply:
Ohh yep, looks like that accidentally got reset somehow